e2e/packer/gcp/main.pkr.hcl

```diff
@@ -98,6 +98,24 @@ source "googlecompute" "nephio-packer" {
 build {
   sources = ["source.googlecompute.nephio-packer"]
 
+  # ------------------------
+  # Fix apt / command-not-found issue
+  # ------------------------
+  provisioner "shell" {
+    inline = [
+      "echo '=============================================='",
+      "echo 'Fixing apt command-not-found issue...'",
+      "echo '=============================================='",
+      "sudo rm -rf /var/lib/apt/lists/*",
```

Comment:
Isn't the order reverse? I would say it should be like this 

```
      "sudo DEBIAN_FRONTEND=noninteractive apt-get update || true",
      "sudo DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends apt-utils || true"
      "sudo rm -rf /var/lib/apt/lists/*",
      "sudo apt-get clean"
```

First update, then clean the cache and remove the lists.

Reply:
I suppose it is correct in this case because of the error described in --> https://storage.googleapis.com/prow-nephio-sig-release/logs/pre-bake-daily-ubuntu-jammy/1958680734440886272/build-log.txt.